### PR TITLE
[darwinia-ethereum] friendly interaction 

### PR DESCRIPTION
**Pull request by fewensa**
same modify with #237  , for darwinia-ethereum bridge

### CODE REVIEW

**Comment by fewensa** [comment]
We can merge this first. redeem still has bugs, wait for https://github.com/darwinia-network/bridger/pull/249 merge then fix in darwinia-ethereum

File: task/task-darwinia-ethereum/src/service/ethereum/mod.rs [hunk 1]
```diff
@@ -56,104 +57,82 @@ fn get_topics_list(ethereum_config: EthereumConfig) -> Vec<(H160, Vec<H256>)> {
...
-                    ToEthereumMessage::Restart(force) => {
-                        if force {
-                            is_started = false;
-                        }
-                        let mut cloned_sender_to_ethereum = sender_to_ethereum.clone();
-                        cloned_sender_to_ethereum
-                            .send(ToEthereumMessage::Start)
-                            .await?;
-                    }
-                    _ => continue,
```
Comment (HackFisher) [inline task/task-darwinia-ethereum/src/service/ethereum/mod.rs:117]: What's the new restart mechanism if this is deleted?

Comment (fewensa) [reply task/task-darwinia-ethereum/src/service/ethereum/mod.rs:117]: The darwinia/ethereum scan service will auto start with task. all code run in a loop
https://github.com/darwinia-network/bridger/blob/da5fdeeb1b3fd24b8b40eff8a956432ad68ac899/task/task-darwinia-ethereum/src/service/ethereum/mod.rs#L105-L123
when happen an error, will start again

Comment (fewensa) [reply task/task-darwinia-ethereum/src/service/ethereum/mod.rs:117]: Also we can use the kv command to control service state

start darwinia/ethereum scan service
```bash
# set darwinia
./target/release/bridger kv -n task-darwinia-ethereum put scan.darwinia.running true
# set ethereum
./target/release/bridger kv -n task-darwinia-ethereum put scan.ethereum.running true
```

stop darwinia/ethereum scan service
```bash
# set darwinia
./target/release/bridger kv -n task-darwinia-ethereum put scan.darwinia.running false
# set ethereum
./target/release/bridger kv -n task-darwinia-ethereum put scan.ethereum.running false
```

**Review by HackFisher — APPROVED** [review]

**Pull request merged**